bot/extensions/levelling/commands.py

```diff
@@ -78,6 +87,10 @@ async def cog_load(self):
         for lvl in range(101):
             xp = 5 * (lvl**2) + (50 * lvl) + 100
             self.required_xp.append(xp + self.required_xp[-1])
+        config = await Config.ensure_exists(guild_id=self.bot.guild.id)
+        self.xp_boost = config.xp_boost
+        self.min_xp = config.min_xp
+        self.max_xp = config.max_xp
```

Comment:
Not possible as we have one config per guild.
The idea is that any guild should be able to add the bot and configure it for themselves